docs/book/src/developers/jobs.md

```diff
@@ -0,0 +1,102 @@
+# Jobs
+
+This document provides an overview of our jobs running via Prow and Github actions.
+
+## Builds and Tests running on the default branch
+
+
+ > NOTE: To see which test jobs execute which e2e tests, you can click on the links which lead to the
+  respective test overviews in testgrid.
+
+
+### Legend
+
+    🟢 REQUIRED - Jobs that have to run successfully to get the PR merged.
+
+
+### Presubmits
+
+  Prow Presubmits:
+
+  - 🟢 [pull-cluster-api-provider-azure-build]  `./scripts/ci-build.sh`
+  - 🟢 [pull-cluster-api-provider-azure-test]  `./scripts/ci-test.sh` 
+  - 🟢 [pull-cluster-api-provider-azure-e2e] ` ./scripts/ci-e2e.sh`
+       * GINKGO_FOCUS:`Workload cluster creation` GINKGO_SKIP:`Creating a GPU-enabled cluster|.*Windows.*|.*AKS.*|Creating a cluster that uses the external cloud provider`
```

Comment:
let's make this a runnable command instead so people can copy and past it to run locally, ie:

```
[pull-cluster-api-provider-azure-e2e] `GINKGO_FOCUS="Workload cluster creation" GINKGO_SKIP="Creating a GPU-enabled cluster|.*Windows.*|.*AKS.*|Creating a cluster that uses the external cloud provider" ./scripts/ci-e2e.sh` 
```

Same with the other ones below